client/dist/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+	<head>
+	  <title>Shop Summary</title>
+	</head>
+	<body>
+		<div id="app"></div>
```

Comment:
We might all have an id of app, you might want to change it to something more specific